### PR TITLE
Mocking

### PR DESCRIPTION
**Pull request by AlexGalichenko**

### CODE REVIEW

File: src/mock.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+ * @param {string} memoryKey - memory key to store mock instance
+ * @example When I create mock for '/yourservice/**' as 'mock1'
+ * @example When I create mock for '$mockUrlTemplate' as 'mock1'
+ */
+When('I create mock for {string} as {string}', async function (urlTemplate: string, memoryKey: string) {
+    const url = await getValue(urlTemplate);
+    memory.setValue(memoryKey, url);
+});
+
+async function respondWith(mockKey: string, statusCode: string, body: string) {
```
Comment (KirillBogdanets) [inline src/mock.ts:18]: please add returned type

Comment (AlexGalichenko) [reply src/mock.ts:18]: It's void. I think can be skipped.

Comment (KirillBogdanets) [reply src/mock.ts:18]: yep, it's could but it will be an error for linters and code smell for sonar

Comment (AlexGalichenko) [reply src/mock.ts]: added

**Review by KirillBogdanets — APPROVED** [review]

**Pull request merged**